straxen/plugins/event_processing.py

```diff
@@ -643,15 +618,46 @@ class CorrectedAreas(strax.Plugin):
     Note:
         Please be aware that for both, the main and alternative S1, the
         area is corrected according to the xy-position of the main S2.
-        
+
         There are now 3 components of cS2s: cs2_top, cS2_bottom and cs2.
         cs2_top and cs2_bottom are corrected by the corresponding maps,
         and cs2 is the sum of the two.
     """
-    __version__ = '0.1.3'
+    __version__ = '0.2.0'
 
     depends_on = ['event_basics', 'event_positions']
 
+    # Descriptor configs
+    # online elife
+    elife = straxen.URLConfig(default='cmt://elife?version=ONLINE&run_id=plugin.run_id')
+
+    # online s1_xyz_map
+    s1_map = straxen.URLConfig(default=f'interpolatingmap://resource://cmt://s1_xyz_map_{DEFAULT_POSREC_ALGO}?'
+                                       'version=ONLINE'
+                                       '&run_id=plugin.run_id'
+                                       '&fmt=json')
+
+    # online s2_xy map
+    s2_map = straxen.URLConfig(default=f'interpolatingmap://resource://cmt://s2_xy_map_{DEFAULT_POSREC_ALGO}?'
+                                       'version=ONLINE'
+                                       '&run_id=plugin.run_id'
+                                       '&fmt=json')
```

Comment:
Not sure how long it takes to load an interpolating map, but if its not super quick I would recommend adding `cache=True` or `cache=MAXSIZE` here (where MAXSIZE is an integer defining the maximum maps you want it to cache)

Reply:
Ah I thought caching was turned on by default. So just add this to the url string like so? `.'..&cache=true'` ?

Reply:
I guess caching by default wouldnt be a terrible idea but I'm always hesitant about introducing caching, debugging cache related issues can be a nightmare... You need to pass it as a kwarg to URLConfig, so `URLConfig(..., cache=True)`.